projects/igniteui-angular/src/lib/test-utils/grid-functions.spec.ts

```diff
@@ -918,6 +918,13 @@ export class GridFunctions {
         return conditionChips[index];
     }
 
+    public static getAllFilterConditionChips(fix) {
```

Comment:
Make the `getFilterConditionChip` method to call this one to avoid code duplication.